### DISCUSSION TITLE
Not showing via Custom Components

### DISCUSSION THREAD

**Discussion by digitalurban** [tolwi/hassio-ecoflow-cloud#103] [General]
This may well be a stupid question - but how to i install (?) I have gone the normal route via custom components in HACS but it does not show up in my install list? Am i missing something fundermental, any help would be great as i have a Delta Max 2 and it would be great to get the data in HA.

Andy

**Comment by danroth77** [comment]
Have you been able to install in the meantime?

**Reply by digitalurban** [reply]
A reboot sorted it - thanks for the reply :)

**Reply by horsey2022** [reply]
I can't get HACS to show this ecoflow component in the HACS Integrations
I have rebooted HACS and the whole HA.
Can you tell me what you put and into which directory.
I have tried putting the whole hassio-ecoflow-cloud-main in /root/config/custom_components 
I also tried putting the ecoflow_cloud directory in /root/config/custom_components

**Reply by tolwi** [reply]
you have to add a custom repository: HACS - Settings (three dots) - Custom Repositories .. add "https://github.com/tolwi/hassio-ecoflow-cloud"

**Reply by horsey2022** [reply]
Thanks, this worked.

**Reply by tronron** [reply]
As of March 2024 does this integration still work?

**Reply by Nid01** [reply]
@tronron 
> As of March 2024 does this integration still work?

As far as I know: Yes, it still works?
Where lies the origin of your question?
Are you having any problems?

**Reply by tronron** [reply]
I had a similar integration some time ago, and Ecoflow closed the ports it used with a firmware update, so it stopped working... I did get this one to work, thank you!
